### PR TITLE
Update the build badge

### PR DESCRIPTION
**Pull request by ialidzhikov**
**What this PR does / why we need it**:
Update the build badge

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
NONE
```

### CODE REVIEW

File: pkg/driver/driver_azure.go [hunk 1]
```diff
@@ -167,7 +167,7 @@ func (d *AzureDriver) getVMParameters(vmName string, networkInterfaceReferenceID
 		VMParameters.Identity = &compute.VirtualMachineIdentity{
 			Type: compute.ResourceIdentityTypeUserAssigned,
 			UserAssignedIdentities: map[string]*compute.VirtualMachineIdentityUserAssignedIdentitiesValue{
-				*d.AzureMachineClass.Spec.Properties.IdentityID: &compute.VirtualMachineIdentityUserAssignedIdentitiesValue{},
+				*d.AzureMachineClass.Spec.Properties.IdentityID: {},
```
Comment (prashanth26) [inline pkg/driver/driver_azure.go]: Have you tested this change? Does it work? And why is it needed? I ask this for my understanding.

File: vendor/k8s.io/klog/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 klog
 ====
 
-klog is a permanent fork of https://github.com/golang/klog.
+klog is a permanent fork of https://github.com/golang/glog.
```
Comment (prashanth26) [inline vendor/k8s.io/klog/README.md]: Why has the vendor file changed here? We haven't re-vendored in this commit have we?

File: pkg/driver/driver_azure.go (hunk 1 above)
Comment (ialidzhikov) [reply pkg/driver/driver_azure.go]: The -s flag (simplified) of gofmt is preferred. Ref https://golang.org/cmd/gofmt/#hdr-The_simplify_command
```
gofmt -l -w -s pkg/driver/driver_azure.go
```

File: vendor/k8s.io/klog/README.md (hunk 2 above)
Comment (ialidzhikov) [reply vendor/k8s.io/klog/README.md]: I just used `make revendor`.

**Comment by prashanth26** [comment]
Okay. Thanks for that.

**Pull request merged**